pkg/cmd/pulumi/stack_import.go

```diff
@@ -122,12 +122,14 @@ func saveSnapshot(ctx context.Context, s backend.Stack, snapshot *deploy.Snapsho
 		}
 	}
 	// Validate the stack. If --force was passed, issue an error if validation fails. Otherwise, issue a warning.
-	if err := snapshot.VerifyIntegrity(); err != nil {
-		msg := fmt.Sprintf("state file contains errors: %v", err)
-		if force {
-			cmdutil.Diag().Warningf(diag.Message("", msg))
-		} else {
-			result = multierror.Append(result, errors.New(msg))
+	if !backend.DisableIntegrityChecking {
```

Comment:
Not actually sure if we should check `DisableIntegrityChecking` here given we've also already got the force option.

Reply:
I think we still should.  It would be kind of confusing to have to remember to use a `--force` option here when `--disable-integrity-checking` doesn't work.